### DISCUSSION TITLE
Inserting link to specific page of pdf(URI)

### DISCUSSION THREAD

**Discussion by aarthihWN** [pymupdf/PyMuPDF#2160] [Looking for help]
Hello,
I have a requirement to add a link to a pdf so that it opens another pdf on the internet on a specific page. Below is the sample code of my trial. Although the page number is mentioned, the document always opens on the first page. 

link = {
"is_external" : True,
"kind" : fitz.LINK_URI,
"from" : rect[0],
"page" : 3,
"uri" : link to my pdf 
}

The pymupdf documentation mentions '_The page number (in this or the target document) this destination points to. Only set if [linkDest.kind](https://pymupdf.readthedocs.io/en/latest/linkdest.html#linkDest.kind) is [LINK_GOTOR]_'

So for LINK_URI, where do I set the target page number? 
If I specify '#page=pagenumber' with the URI link, it does open the document on the specified page . Is this the only way?

Thanks in advance for your help!

**Comment by JorjMcKie** [comment]
> If I specify '#page=pagenumber' with the URI link, it does open the document on the specified page . Is this the only way?

Currently yes. But actually a good idea to include it in the "official" API.
Could you please submit a respective enhancement request?
Thank you!

**Reply by aarthihWN** [reply]
ok .Thanks for your quick response. 
I misunderstood your comments here thinking this was possible. 
https://stackoverflow.com/questions/73071411/how-to-add-a-link-with-python-in-one-pdf-to-another-pdf.

So how do I submit the  request for this? Issues -> New feature?
